host_vars/deploy/main.yml

```diff
@@ -39,9 +39,7 @@ static_ngi_log_path: "{{ static_ngi_pipeline_path }}/log"
 ngi_softlinks: "{{ ngi_pipeline_workdir }}/softlinks"
 
 NGI_venv_name: "NGI"
-NGI_venv_py2_name: "NGI_py2"
 ngi_pipeline_venv: "{{ sw_path }}/anaconda/envs/{{ NGI_venv_name }}"
```

Comment:
Maybe `{{ sw_path }}/anaconda` could be changed to `{{ anaconda_path }}` so that it's a single source?

Reply:
Good point :+1: